trento/xml/article_sap_trento.xml

```diff
@@ -213,6 +213,10 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         two CPU cores and 64&nbsp;GB of storage.
         When using K3s, such storage should be provided under <filename>/var/lib/rancher/k3s</filename>.
       </para>
+     <para>
+      Trento is powered by event driven technology. Registered events are stored in a postgresql database with a retention
+      period of 10 days. For each host registered with Trento, you need to allocate at least 1.5GB of space in the postgresql database.
```

Comment:
As postgresql is a unique name, it should be written with an uppercase "P". Additionally it has a specific spelling and we already have an entity for that. I'd recommend to use this:

```suggestion
      Trento is powered by event driven technology. Registered events are stored in a  &postgresql; database with a retention
      period of 10 days. For each host registered with Trento, you need to allocate at least 1.5GB of space in the &postgresql; database.
```